server/verify_tdx.go

```diff
@@ -19,7 +18,7 @@ func TdxDefaultOptions() *VerifyTdxOpts {
 }
 
 // VerifyTdxAttestation checks that the TDX attestation quote is valid
-func VerifyTdxAttestation(attestation *tpb.QuoteV4, opts *VerifyTdxOpts) error {
+func VerifyTdxAttestation(attestation any, opts *VerifyTdxOpts) error {
```

Comment:
Making API future compatible with other quote formats.

Reply:
This (any) can be confused for people to call this function, could you add more description of "attestation" and potentially how to generate an "attestation"

Reply:
Added description.